_playwright-tests/UI/Templates.spec.ts

```diff
@@ -8,11 +8,27 @@ test.describe('Templates', () => {
   }) => {
     await navigateToTemplates(page);
     await closePopupsIfExist(page);
-
-    const AddButton = page.locator('[data-ouia-component-id="create_content_template"]');
-
-    // Wait for the Add button to become enabled (up to 10 seconds)
-    await AddButton.first().isEnabled({ timeout: 10000 });
+    await page.getByRole('button', { name: 'Add content template' }).click();
+    await page.getByRole('button', { name: 'Select architecture' }).click();
+    await page.getByRole('option', { name: 'x86_64' }).click();
+    await page.getByRole('button', { name: 'Select version' }).click();
+    await page.getByRole('option', { name: 'el8' }).click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Add the step to select 'Red hat repos Checkbox'
+    await page.getByRole('button', { name: 'Next' }).click();
+    // Add the step to select 'checkbox for cutom repo'
+    await page.getByRole('button', { name: 'Next' }).click();
+    await page.locator('input[name="use-latest-snapshot"]').click();
+    await page.getByRole('button', { name: 'Next' }).click();
+    const nameInput = page.getByPlaceholder('Enter name');
+    // const descriptionInput = page.getByPlaceholder('Enter Description');
+    await nameInput.fill('demo_template');
```

Comment:
lines 23 and 25 can be inlined and you don't need to declare the `nameInput` variable as it's only used once 😅